File: Neomer.EveryPrice.REST.Web/Models/TagViewModel.cs
using Neomer.EveryPrice.SDK.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Neomer.EveryPrice.REST.Web.Models
{

    [Serializable]
    public class TagViewModel
    {
        public TagViewModel()
        {

        }

        public TagViewModel(ITag tag)
        {
            Uid = tag.Uid;
            Value = tag.Value;
            EntityCount = tag.Shops != null ? tag.Shops.Count : 0;
        }

        public Guid Uid;

        public String Value;

        public int EntityCount;

    }
}

File: Neomer.EveryPrice.Android/app/src/main/java/com/neomer/everyprice/api/models/TagViewModel.java
package com.neomer.everyprice.api.models;

import java.util.UUID;

public class TagViewModel {

    private UUID Uid;
    private String Value;
    private int EntityCount;

    public UUID getUid() {
        return Uid;
    }

    public void setUid(UUID uid) {
        Uid = uid;
    }

    public String getValue() {
        return Value;
    }

    public void setValue(String value) {
        Value = value;
    }

    public int getEntityCount() {
        return EntityCount;
    }

    public Tag toTag() {
        return new Tag(Uid, Value);
    }

}
